### DISCUSSION TITLE
How to retrieve client identifier by React code in React Native project.

### DISCUSSION THREAD

**Discussion by GuoYongming** [invertase/react-native-firebase#5443] [Q&A]
There is a new requirement in our project that retrieve client identifier, our project is a React Native project, we checked some documents, such as [https://firebase.google.com/docs/projects/manage-installations#retrieve_client_identifer](https://firebase.google.com/docs/projects/manage-installations#retrieve_client_identifer) and [https://rnfirebase.io/](https://rnfirebase.io/), but they do not mention any methods how to retrieve client identifier with React code.
Is there another way to release this function? i am looking forward to your help. thank you!

**Comment by mikehardy** [comment]
I just noticed this sorry. It was answered in the duplicate issue you filed a bit after, I believe? https://github.com/invertase/react-native-firebase/issues/5445#issuecomment-865468737
